ui/src/pages/ConcurrencyLimits.vue

```diff
@@ -1,19 +1,55 @@
 <template>
   <p-layout-default class="concurrency-limits">
     <template #header>
-      <PageHeading :crumbs="[{ text: 'Task Run Concurrency Limits' }]">
-        <template #after-crumbs>
-          <p-button small icon="PlusIcon" @click="open" />
-        </template>
-      </PageHeading>
-      <ConcurrencyLimitsCreateModal v-model:showModal="showModal" />
+      <PageHeading :crumbs="[{ text: 'Concurrency' }]" />
     </template>
-    <ConcurrencyLimitsTable />
+    <p-tabs v-model:selected="tab" :tabs="tabs">
+      <template #global>
+        <PageHeading size="lg" :crumbs="[{ text: 'Global Concurrency Limits' }]">
+          <template #after-crumbs>
+            <p-button small icon="PlusIcon" @click="openGlobal" />
+          </template>
+        </PageHeading>
+        <ConcurrencyLimitsV2CreateModal v-model:showModal="showModalGlobal" />
+        <ConcurrencyLimitsV2Table />
+      </template>
+      <template #task-run>
+        <PageHeading size="lg" :crumbs="[{ text: 'Task Run Concurrency Limits' }]">
+          <template #after-crumbs>
+            <p-button small icon="PlusIcon" @click="openTaskRun" />
+          </template>
+        </PageHeading>
+        <ConcurrencyLimitsCreateModal v-model:showModal="showModalTaskRun" />
+        <ConcurrencyLimitsTable class="concurrency-limits__task-limits-table" />
+      </template>
+    </p-tabs>
   </p-layout-default>
 </template>
 
   <script lang="ts" setup>
-  import { PageHeading, ConcurrencyLimitsCreateModal, ConcurrencyLimitsTable, useShowModal } from '@prefecthq/prefect-ui-library'
+  import { PageHeading, ConcurrencyLimitsV2Table, ConcurrencyLimitsTable, ConcurrencyLimitsCreateModal, ConcurrencyLimitsV2CreateModal, useShowModal, useTabs } from '@prefecthq/prefect-ui-library'
+  import { computed } from 'vue'
+  
+  const { showModal: showModalGlobal, open: openGlobal } = useShowModal()
+  const { showModal: showModalTaskRun, open: openTaskRun } = useShowModal()
+
+  const computedTabs = computed(() => [
+    { label: 'Global' },
+    { label: 'Task Run' },
+  ])
+
+  const { tab, tabs } = useTabs(computedTabs)
```

Comment:
Also you don't actually need to use this composition unless you have dynamic tabs.